public_html/application/classes/Controller/Barracks.php

```diff
@@ -73,11 +73,10 @@ function manageprisoners($structure_id)
 		}
 
 		$prisoners = ORM::factory("character_sentence")
-			->where(array(
-				'prison_id' => $structure_id,
-				'imprisonment_start is not' => null,
-				'status' => 'executing'
-			))->find_all();
+			->where('prison_id', '=', $structure_id)
+			->and_where('imprisonment_start is not', 'is not', null)
```

Comment:
`->and_where('imprisonment_start', 'is not', null)`

Need to drop the the 'is not' from the string in the first param, it's just the column name in this version, whereas in the previous version they just interpolated it straight into a query so you could put operations in the string to change the behavior.